.github/workflows/validate-quick-start-module.yml

```diff
@@ -0,0 +1,30 @@
+name: Validate quick start module
+on:
+  pull_request:
+    paths:
+      - published_versions.json
+      - /assets/quick-start-module.js
+      - .github/workflows/validate-quick-start-module.yml
```

Comment:
Why it needs to be run when those files are modified? As workflow is implemented right now, it does not depend on any of those files

Reply:
Validate workflow will run on the autogenerated PR. And autogenerated PR should have published_versions.json and quick_start module in list of files